CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.3.0] - 2023-04-26
+
+### Features
+
+- Make cloudproof crate publishable and publish it in CI on tags
```

Comment:
Why there is not the rexport of crypto_core here? (neither inside the changed files…)

Reply:
There was a 1.2.0 released yesterday which contains the reexport feature